crates/viewer/re_viewer/src/app.rs

```diff
@@ -579,15 +577,23 @@ impl App {
             }
             SystemCommand::AddRedapServer { endpoint } => {
                 let re_uri::CatalogEndpoint { origin } = endpoint;
-                self.state.redap_servers.add_server(origin);
+                self.state.redap_servers.add_server(origin.clone());
+
+                if self
+                    .store_hub
+                    .as_ref()
+                    .map_or(true, |store_hub| store_hub.active_entry().is_none())
```

Comment:
I usually find `is_none_or` and `is_some_and` more readable

```suggestion
                    .is_none_or(|store_hub| store_hub.active_entry().is_none())
```

Reply:
Wonder why this is not picked up by [`unnecessary_unwrap_or`](https://rust-lang.github.io/rust-clippy/master/index.html#unnecessary_map_or).

Reply:
Good question 🤔